tests/unit/support/response.json

```diff
@@ -0,0 +1,159 @@
+        },
+        "body": "Merge pull request #1 from Jdmcc890/Hazards",
+        "score": 1.0
+      },
+      {
+        "url": "https://api.github.com/repos/learn-co-students/hs-zhw-shoes-layout-cb-000/issues/177",
+        "repository_url": "https://api.github.com/repos/learn-co-students/hs-zhw-shoes-layout-cb-000",
+        "labels_url": "https://api.github.com/repos/learn-co-students/hs-zhw-shoes-layout-cb-000/issues/177/labels{/name}",
+        "comments_url": "https://api.github.com/repos/learn-co-students/hs-zhw-shoes-layout-cb-000/issues/177/comments",
+        "events_url": "https://api.github.com/repos/learn-co-students/hs-zhw-shoes-layout-cb-000/issues/177/events",
+        "html_url": "https://github.com/learn-co-students/hs-zhw-shoes-layout-cb-000/pull/177",
+        "id": 500585119,
+        "node_id": "MDExOlB1bGxSZXF1ZXN0MzIzMDIzMDE2",
+        "number": 177,
+        "title": "Done",
+        "user": {
+          "login": "rowlandjl",
+          "id": 14290075,
+          "node_id": "MDQ6VXNlcjE0MjkwMDc1",
+          "avatar_url": "https://avatars3.githubusercontent.com/u/14290075?v=4",
+          "gravatar_id": "",
+          "url": "https://api.github.com/users/rowlandjl",
+          "html_url": "https://github.com/rowlandjl",
+          "followers_url": "https://api.github.com/users/rowlandjl/followers",
+          "following_url": "https://api.github.com/users/rowlandjl/following{/other_user}",
+          "gists_url": "https://api.github.com/users/rowlandjl/gists{/gist_id}",
+          "starred_url": "https://api.github.com/users/rowlandjl/starred{/owner}{/repo}",
+          "subscriptions_url": "https://api.github.com/users/rowlandjl/subscriptions",
+          "organizations_url": "https://api.github.com/users/rowlandjl/orgs",
+          "repos_url": "https://api.github.com/users/rowlandjl/repos",
+          "events_url": "https://api.github.com/users/rowlandjl/events{/privacy}",
+          "received_events_url": "https://api.github.com/users/rowlandjl/received_events",
+          "type": "User",
+          "site_admin": false
+        },
+        "labels": [],
+        "state": "open",
+        "locked": false,
+        "assignee": null,
+        "assignees": [],
+        "milestone": null,
+        "comments": 0,
+        "created_at": "2019-10-01T00:00:04Z",
+        "updated_at": "2019-10-01T00:00:04Z",
+        "closed_at": null,
+        "author_association": "NONE",
+        "pull_request": {
+          "url": "https://api.github.com/repos/learn-co-students/hs-zhw-shoes-layout-cb-000/pulls/177",
+          "html_url": "https://github.com/learn-co-students/hs-zhw-shoes-layout-cb-000/pull/177",
+          "diff_url": "https://github.com/learn-co-students/hs-zhw-shoes-layout-cb-000/pull/177.diff",
+          "patch_url": "https://github.com/learn-co-students/hs-zhw-shoes-layout-cb-000/pull/177.patch"
+        },
+        "body": null,
+        "score": 1.0
+      }
+    ]
+}
```

Comment:
Missing newline at the end.

It is a good advice to use and [.editorconfig](https://editorconfig.org/) plugin. ;)